SmartSearch.LuceneNet/Internals/SpecializedFields/SpecializedFieldSpecifications.cs

```diff
@@ -0,0 +1,27 @@
+﻿using SmartSearch.Abstractions;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SmartSearch.LuceneNet.Internals.SpecializedFields
+{
+    internal class SpecializedFieldSpecifications
```

Comment:
This class was originally in SmartSearch.LuceneNet/Internals/SpecializedFields/ISpecializedFieldSpecification.cs